File: python3_solution_set2/2643. Row With Maximum Ones.py
class Solution:
    def rowAndMaximumOnes(self, mat: List[List[int]]) -> List[int]:
        ans = 0
        maxs = 0

        for indx, r in enumerate(mat):
            c = r.count(1)
            if maxs < c:
                maxs = c
                ans = indx


        return [ans, maxs]

File: go_solution_set1/2643. Row With Maximum Ones.go
func rowAndMaximumOnes(mat [][]int) []int {
    ans := []int {}
    maxs := -1
    for indx,val := range mat{
        cnt := count_one(val)
        if cnt > maxs {
            ans = append(ans, indx)
            ans = append(ans, cnt)
            maxs = cnt
        }
    }
    n := len(ans)
    return []int{ans[n-2], ans[n-1]}
}

func count_one(arr []int) int{
    result := 0

    for _,val := range arr{
        if val == 1{
            result++
        }

    }
    return result
}
